src/content/1.7/faq/tips-and-tricks/hook-tips.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Hook tips
+---
+
+# Hook tips
+
+Let's say we are need to add some javascript to the 'Order create' BO page. So we use `hookActionAdminControllerSetMedia`.
+This hook is called on every BO page, so we need to filter the executions.
+```php
+if ('AdminOrders' === Tools::getValue('controller')) {
+    // now we know we are only applying changes to admin order pages,
+    // but we aren't yet sure if it's a order create or order view page
+}
+```
+
+In case we are using Prestashop version lower than 1.7.7.0, then Order page is not yet migrated to Symfony framework
+and we can check if it has following query parameters
```

Comment:
```suggestion
In case we are using Prestashop version lower than 1.7.7.0, then Order page is not yet migrated to Symfony framework and we can check if it has following query parameters:
```